plenum/test/batching_3pc/catch-up/test_catchup_during_3pc.py

```diff
@@ -0,0 +1,30 @@
+import pytest
```

Comment:
Well, yes, I think it's possible, but not sure that the tests will be totally predictable in this case, since the condition I would like to simulate is quite specific.